locales/en/connection-tab.json

```diff
@@ -0,0 +1,29 @@
+{
+  "admin_url_button_aria_label": "More information about the URL for Kafka Admin REST API",
```

Comment:
@wtrocki Following up on our chat offline, do you know what the updated name for this one will be? Is it definitely changing from **Kafka Admin REST API** to **Kafka Instance API** or is that still TBD?

Reply:
I will create PR for Kafka Instance but that will be subjective to PR being approved.
Kafka, Connectors and Registry management were approved so now I will look into dataplane names